spec/std/big/big_float_spec.cr

```diff
@@ -2,6 +2,67 @@ require "spec"
 require "big_float"
 
 describe "BigFloat" do
+  describe "new" do
+    bsi = "123456789012345678901"
```

Comment:
How about `int_string`, and `int_bigfloat` etc. The current ones are just as unreadable.